specs-derive/src/lib.rs

```diff
@@ -15,7 +15,7 @@ extern crate syn;
 use proc_macro::TokenStream;
 use syn::{
     parse::{Parse, ParseStream, Result},
-    DeriveInput, Path,
+    DeriveInput, Path, PathArguments,
 };
 
 mod impl_saveload;
```

Comment:
Can you add an example onto the derive macro docs below demonstrating the new usage that this enables and noting how it differs from the case where `<Self>` is assumed?

Reply:
I misunderstood I think.
007d5bb7cbeb77a0fcaf6386708b86389e9d4d96 adds (maybe unnecessary) Info to the tutorial.
fd59c30ba7daa86fe3ed30ae75d9943ff3665329 adds info to the docs.

Reply:
I just meant the docs, but the other addition looks fine to me! Sorry, I would have put my comment directly on the section where the docs are but GitHub's review doesn't allow that.